src/app/shared/menu/date/custom-date-range-dropdown.component.spec.ts

```diff
@@ -20,7 +20,7 @@ describe('CustomDateRangeDropdownComponent', () => {
   let el: HTMLElement;
   let store: Store<any>;
 
-  const routerState = {
+  const routerParams = {
```

Comment:
no better time to rename a variable than when you're changing 2000 other lines in an app 😬